test/test_transformers.py

```diff
@@ -1292,6 +1292,19 @@ def test_unaligned_tensors(self):
             self.assertRaises(RuntimeError, lambda: torch.nn.functional._scaled_dot_product_attention(
                 q, k, v, None, 0.0, False, False))
 
+    @unittest.skipIf(not TEST_CUDA or not SM80OrLater or TEST_WITH_ROCM, "CUDA unavailable")
+    def test_flash_fail_fp32t(self):
+        device = 'cuda'
+        dtype = torch.float
+        size = (2, 2, 8, 5)
```

Comment:
Flash doesn't accept head dim size of 5 so I don't this is failing for dtype 32 which we do already test with. If you decorate with auto cast does this enable flash?